src/Controllers/AuthController.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace Railroad\MusoraApi\Controllers;
+
+use Carbon\Carbon;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Railroad\Ecommerce\Entities\Subscription;
+use Railroad\Ecommerce\Events\AppSignupStartedEvent;
+use Railroad\Ecommerce\Repositories\SubscriptionRepository;
+use Railroad\MusoraApi\Contracts\ProductProviderInterface;
+use Railroad\MusoraApi\Contracts\UserProviderInterface;
+use Railroad\Usora\Entities\UserFirebaseTokens;
+use Railroad\Usora\Managers\UsoraEntityManager;
+use Railroad\Usora\Repositories\UserFirebaseTokensRepository;
+use Railroad\Usora\Repositories\UserRepository;
+use Tymon\JWTAuth\JWTAuth;
+
+class AuthController extends Controller
+{
+    /**
+     * @var EntityManager
```

Comment:
There are some missing php use statements and some extra ones not being used. This likely needs some cleanup across the entire project regarding use statements.